authorization/auth_cloudSolutionProvider.md

```diff
@@ -0,0 +1,62 @@
+
+Additionally, as a partner developer, you can build a Partner Managed App (PMA), also known as a *pre-consented* app, to manage your customers' Microsoft services. PMAs are often termed *pre-consented* apps because all your customers are automatically pre-consented for your partner managed apps. This means when a user from one of your customer tenants uses one of your partner managed apps, the user can use it without seeing any consent. And with the DAP model, your partner agents can access customers through your partner managed application.
+
+## How to set-up a Partner Managed Application
+An application is viewed as *Partner Managed* when it is granted elevated permissions to access your customers' data. Note that partner managed apps can *only* be configured on Partner tenants.
+
+The initial steps required here follow most of the same steps used to register and configure a multi-tenant application:
+
+1. Register your application in your Partner tenant. You can use the <a href="https://portal.azure.com" target="_newtab">Azure Portal</a> for this. To function as a PMA, an application must be configured as a multi-tenant app.
+
+2. Configure your multi-tenant app with the *required permissions* it needs using a least privileged approach.
+
+3. Finally grant your Partner Managed App those configured permissions for all your customers. You can so this by adding the **servicePrincipal** that represents the app to the AdminAgents group in your Partner tenant, using Azure AD powershell V2. You can download and install Azure AD PowerShell V2 from <a href="https://www.powershellgallery.com/packages/AzureAD" target="_newtab">here</a>.  Follow these steps to find the AdminAgents group, the **servicePrincipal** and add it to the group.
+
+    1. Open a PowerShell session and connect to your partner tenant by entering your admin credentials into the sign-in window.
+    ```code
+    > Connect-AzureAd
+    ```
+    2. Next find the group that represents the AdminAgents. 
+    ```code
+    > $group = Get-AzureADGroup -Filter "displayName eq 'AdminAgents'"  
+    ```
+    3. Find the service principal that has the same *appId* as your app.
+    ```code
+    > $sp = Get-AzureADServicePrincipal -Filter "appId eq '{yourAppsAppId}'"
+    ```
+    4. Finally add the service principal to the AdminAgents group.
+    ```code
+    > Add-AzureADGroupMember -ObjectId $group.ObjectId -RefObjectId $sp.ObjectId
+    ```
+
+## Token acquisition flows
+Partner managed apps are identical to regular multi-tenant apps in virtually all respects. As such, the token acquisition flows <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-code" target="_newtab">authorization code grant flow</a> or the <a href="https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-service-to-service" target="_newtab">service to service client credentials flow</a> are exactly the same.
```

Comment:
For more concise writing, you can remove the adverbs "virtually" and "exactly" here without changing the meaning.